DiscordBot/Models/DiscordEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DiscordBot.Models
+{
+    class DiscordEvent
```

Comment:
Is this class actually used anywhere? 🙂

Reply:
This class is to establish the structure for which methods in Benji's data access service should return.

Reply:
Okay. For future reference I would in general expect that to be defined when it should be used, e.g. when defining the service class interface. Otherwise it's easy to forget about the class and for it not to be noticed that it's unused 🙂

Reply:
I expect Benji to need the class quite soon, should I leave it or remove it?

Reply:
Fine to leave, just a note for next time 🙂